deno/send_message/README.md

```diff
@@ -0,0 +1,101 @@
+# SendMessage() 
+
+A sample Deno Cloud Function sending a message using a specific channel to a receiver
+
+Supported channels are `SMS`, `EMAIL` ,`DISCORD` and `TWITTER`.
```

Comment:
Supported channels should be `Email`, `SMS`, `Discord`, and `Twitter`

Reply:
It is already the same right?

Reply:
Case is different right? And that matters in your code?

Reply:
yes

Reply:
fixed